src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -67,11 +66,13 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
             final BlockPos position = Settings.instance.getPosition();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "position". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "position" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1481)